packages/fast-components/src/slider-label/slider-label.template.ts

```diff
@@ -0,0 +1,27 @@
+import { html, when } from "@microsoft/fast-element";
+import { SliderLabel } from "./slider-label";
+import { SliderOrientation } from "../slider";
+
+export const SliderLabelTemplate = html<SliderLabel>`
+    <template
+        class=${x =>
+            x.config.orientation === SliderOrientation.horizontal
+                ? "slider-label-horizontal"
+                : "slider-label-vertical"}
+        style=${x => x.positionStyle}
```

Comment:
Hmm style also - this will clobber any inline styles applied by the app author.

Reply:
Affirmative.

Reply:
Will put this one back in class

Reply:
well wait this one is a problem. the reason I moved it up was to get rid of what was referred to as superfluous dom/div, so I guess it was not superfluous after all? I need to position this control using style.

Reply:
Technically, you should be able to use JS to add/remove the styles. You just cant use a style binding on the host element. The binding isn't smart enough to add/remove only the styles you provide without mucking with user styles. But....this still might be something that's better with a wrapping div. I can see there still be issues with user-land conflicts.

Reply:
as you know from offline conversation I got it to work, so style out of host template now.